packages/nimble-components/src/text-field/styles.ts

```diff
@@ -165,6 +179,10 @@ export const styles = css`
         color: ${bodyDisabledFontColor};
     }
 
+    :host([appearance-readonly]) .control[disabled]::placeholder {
+        color: ${controlLabelFontColor};
+    }
```

Comment:
These aren't intentionally targetting the control vs host right? For consistency can we change this and the selector above to key off state from the host instead of the `.control`?

```css
    :host([disabled]) .control::placeholder {
        color: ${bodyDisabledFontColor};
    }

    :host([disabled][appearance-readonly]) .control::placeholder {
        color: ${controlLabelFontColor};
    }


```